server/src/db/migrations/1_initial-migrations.sql

```diff
@@ -11,5 +11,6 @@ CREATE TABLE "student" (
   student_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   first_name TEXT NOT NULL,
   last_name TEXT NOT NULL,
+  email TEXT NOT NULL UNIQUE,
```

Comment:
Can you please also update the tests to include this field, and add new test cases (e.g. error handling on duplicate emails)

Reply:
I'll look into it. Thanks for your time, Anup!